common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -350,7 +426,7 @@ public AccountRecord getAccount(@Nullable final String environment,
                 mAccountCredentialCache.getCredentialsFilteredBy(
                         null, // homeAccountId
                         environment,
-                        CredentialType.RefreshToken,
+                        CredentialType.IdToken,
```

Comment:
Note to self: need to get this change blessed.

Since broker may be hosting AT/RT pairs for the current Account, we can no longer define the presence of an Account based on whether or not we have an RT for it. Instead, we will say we have an Account if we have an IdToken for it which matches the current apps' client Id.

Reply:
@shoatman @kreedula @heidijinxujia Let's plan to make a follow-up item out of this in tomorrow's scrum